custom_dc/README.md

```diff
@@ -1,499 +1,3 @@
-[homepage.html](../server/templates/custom_dc/custom/homepage.html).
-
-### Update CSS and Javascript
-
-The custom Data Commons provides an
-[overrides.css](../static/custom_dc/custom/overrides.css)
-to override CSS styles. It has a default color override. More style changes can
-be added in that file.
-
-If there are already existing CSS and Javascript files, put them under the
-[/static/custom_dc/custom](../static/custom_dc/custom)
-folder. Then include these files in the `<head>` section of the corresponding
-html files as
-
-```html
-<link href="/custom_dc/custom/<additional>.css" rel="stylesheet" />
-```
-
-or
-
-```html
-<script src="/custom_dc/custom/<additional>.js"></script>
-```
-
-## Environment Variables
-
-Custom Data Commons Development and deployment require a set of environment
-variables to be set. These variables should be set in a text file. Example file
-for SQLite based instance can be found in ["sqlite_env.list"](./sqlite_env.list).
-Example file for CloudSQL based intance can be found in
-["cloudsql_env.list"](./cloudsql_env.list).
-
-Below is a detailed description of all the variables available.
-
-| Variable             | Description                                                                                      |
-| -------------------- | ------------------------------------------------------------------------------------------------ |
-| FLASK_ENV            | Base folder name for custom html and css files, default to 'custom'                              |
-| DC_API_ROOT          | API endpoints to access base Data Commons data                                                   |
-| DC_API_KEY           | API key for accessing base Data Commons API                                                      |
-| GCS_DATA_PATH        | When storing data in Google Cloud Storage, the data path in the form of `gs://<bucket>/<folder>` |
-| USE_SQLITE           | When set to `true`, use local SQLITE as database                                                 |
-| USE_CLOUDSQL         | When set to `true`, use Google Cloud SQL as database                                             |
-| GOOGLE_CLOUD_PROJECT | [`USE_CLOUDSQL=true`] GCP project of the Cloud SQL when                                          |
-| CLOUDSQL_INSTANCE    | [`USE_CLOUDSQL=true`] In the form of `<project_id>:<region>:<instance_id>`                       |
-| DB_USER              | [`USE_CLOUDSQL=true`]Cloud SQL database user                                                     |
-| DB_PASS              | [`USE_CLOUDSQL=true`]Cloud SQL database password                                                 |
-| ADMIN_SECRET         | [Optional] Secret token to perform /admin page operation                                         |
-| MAPS_API_KEY         | [Optional] Used for map visulization place search                                                |
+The contents of this page have moved to https://docs.datacommons.org/custom_dc/.
```

Comment:
Can you verify (and adjust if necessary) that the period at the end is not included in the linked URL.

Reply:
Yes, I had already tested it.